backend/capellacollab/projects/toolmodels/diagrams/routes.py

```diff
@@ -67,14 +67,15 @@ async def get_diagram_metadata(
     )
 
 
-@router.get("/{diagram_uuid}", response_class=fastapi.responses.Response)
+@router.get("/{diagram_file_name}", response_class=fastapi.responses.Response)
```

Comment:
```suggestion
@router.get("/{diagram_uuid_or_filename}", response_class=fastapi.responses.Response)
```

We do still accept and I'd keep it as preferred method. The clients should not care about the fileending.